meson.build

```diff
@@ -0,0 +1,25 @@
+project('shtools', 'c', 'fortran',
+  version: '4.11.0dev0',
+  license: 'BSD-3-Clause',
+  meson_version: '>=1.0.0')
```

Comment:
I think that we should use version 1.1.0+ in order to be compatible with the file `meson.options`.